src/sonic-config-engine/minigraph.py

```diff
@@ -808,7 +823,15 @@ def parse_meta(meta, hname):
                     cloudtype = value
                 elif name == "ResourceType":
                     resource_type = value
-    return syslog_servers, dhcp_servers, ntp_servers, tacacs_servers, mgmt_routes, erspan_dst, deployment_id, region, cloudtype, resource_type
+                elif name == "SwitchId":
```

Comment:
This is configuration is for the Line card host not asic, do we need these attributes ?

Reply:
These are required in the case of single asic linecards. In this case the linecard is a single host device. For multi-asic linecards or multi-asic fixed systems these attributes needs to be parsed for each asic in `parse_asic_meta`.